go/vt/sqlparser/sql.y

```diff
@@ -938,10 +978,157 @@ value_expression:
   {
     $$ = $1
   }
+
 | VALUES openb sql_id closeb
   {
     $$ = &ValuesFuncExpr{Name: $3}
   }
+| CONVERT openb expression ',' convert_type closeb
```

Comment:
I feel like this and below should be a value_expression, as all other things in value_expression reference